svc/GridExportService.js

```diff
@@ -127,17 +128,20 @@ export class GridExportService {
         return {data: headers, depth: 0};
     }
 
-    getRecordRowsRecursive(gridModel, records, columns, depth) {
+    getRecordRowsRecursive(gridModel, recordNodes, columns, depth) {
         const {sortBy, treeMode} = gridModel,
-            sortColIds = sortBy.map(it => it.colId),
+            sortFns = sortBy.map(sort => {
```

Comment:
Could be a fairly concise one-liner: `sortFns = sortBy.map(sort => ({colId}) => ({record}) => record[colId])` though might not be as readable